### PR TITLE
Updates to VSCode and NodeJs-LTS version

### PR DESCRIPTION
**Pull request by jswan106**

### CODE REVIEW

**Comment by didiatworkz** [comment]
git version deprecated #1749 
node.js version deprecated #1756

**Review by twangboy — APPROVED** [review]

**Comment by twangboy** [comment]
Looks like we have some conflicts here

**Comment by jswan106** [comment]
There we go, fixed the conflicts, should be good to go now.

File: vscode.sls [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {% set PROGRAM_FILES = "%ProgramFiles%" %}
 
 vscode:
-  {% for version, guid in (('1.36.1', '2213894ea0415ee8c85c5eea0d0ff81ecc191529'), ('1.37.1', 'f06011ac164ae4dc8e753a3fe7f9549844d15e35'), ('1.38.1', 'b37e54c98e1a74ba89e03073e5a3761284e3ffb0'), ('1.39.1', '88f15d17dca836346e787762685a40bb5cce75a8'), ('1.39.2', '6ab598523be7a800d7f3eb4d92d7ab9a66069390')) %}
+  {% for version, guid in (('1.50.1', 'd2e414d9e4239a252d1ab117bd7067f125afd80a'), ('1.49.3', '2af051012b66169dde0c4dfae3f5ef48f787ff69'), ('1.48.2', 'a0479759d6e9ea56afa657e454193f72aef85bd0'), ('1.39.1', '88f15d17dca836346e787762685a40bb5cce75a8')) %}
```
Comment (twangboy) [inline vscode.sls:8]: We need to keep the older versions around to allow for upgrade and uninstall of those older versions.

**Review by twangboy — CHANGES_REQUESTED** [review]

**Review by twangboy — APPROVED** [review]

**Pull request merged**